student.rb

```diff
@@ -9,4 +11,8 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def add_classroom(classroom)
+    classroom.add_student(self) unless classroom.student.include?(self)
+  end
```

Comment:
- [ OPTIONAL] Great job so far @Rebelzob I will suggest that you define this `add_student` method in this line below to ensure that the `add_classroom` method works as expected :wink: 
```
  classroom.add_student(self) unless classroom.student.include?(self)
```

- Alternatively, you can rewrite the method like this :+1: 
```
  def classroom=(classroom)
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```